doc/tinyramfs.5.scd

```diff
@@ -22,102 +22,110 @@ Options which expect a device as a value can also refer to the device via
 *UUID*, *LABEL*, and *PARTUUID* (eg. LABEL=<label>). This requires a blkid
 utility which supports that identifier.
 
-root
-
-	The device your root filesystem is located on.
-
 hooks
 
 	A comma separated list of hooks to include in the initramfs. Hooks can
 	be found in /lib/tinyramfs/hook.d/.
 
-monolith
+## MOUNTING
 
-	(bool) Instructs tinyramfs whether or not to include kernel modules in
-	the initramfs.
-
-compress
-
-	The utility to use for compressing the initramfs. The option accepts
-	arbitrary commands (eg. 'gzip -9'). Use 'cat' for an uncompressed
-	initramfs.
+root
 
-hostonly
+	The device your root filesystem is located on.
 
-	(bool) Whether or not to only copy modules specific to your system.
+	(available formats: /dev/sdX, LABEL=label, PARTUUID=partuuid, UUID=uuid)
+	
+	Note: using /dev/sdX is not recommeded since device path mappings may change
+	after changing hardware or even updating the kernel.
 
 root_type
 
 	The filesystem you're using for your root device (eg. ext4).
 
-## KEYMAP
+root_opts
 
-keymap_path
+	Arguments passed to mount command when mounting root device.
 
-	Path to your keymap.
+rorw
 
-## LUKS
+	Define root partition ro/rw mounting, default to use ro to make use of init system.
+	
+	(available options: ro, rw)
```

Comment:
rename to `root_readonly` and use it as a boolean value?

Reply:
I would suggest keeping these two params to provide compatibility. A lot of users take well-known parameter for granted.